.github/workflows/templates/build-wheel-release-upload.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     with:
       project: {{ PROJECT/PROJECT_NAME }}
       c_extension: {{ C_EXTENSION/false }}
-      github_admin_username: 'sbillinge'
+      github_admin_username: {{ GITHUB_ADMIN_USERNAME/sbillinge }}
```

Comment:
maybe remove me here also as default (again, it will break everything for us until this is set up in all the projects.

Reply:
done